docs/03_tutorials/tic-tac-toe-game-contract.md

```diff
@@ -293,11 +282,11 @@ void tic_tac_toe::create(const name &challenger, name &host) {
         g.challenger = challenger;
         g.host = host;
         g.turn = host;
-   });
+    });
 }
 ```
 
-### "move" Action Handler 
+### `move` Action Handler 
```

Comment:
ticks "`" in titles

Reply:
Commented above regarding empahsis syntax in headings and titles.